### PR TITLE
[WEB-3127] Get licenses for `/user/{user_id}/licenses`

### PR DESCRIPTION
**Pull request by mshahidi**

### CODE REVIEW

File: src/License/Command/LicenseList.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+declare(strict_types=1);
+
+namespace Serato\SwsSdk\License\Command;
+
+use Serato\SwsSdk\CommandBasicAuth;
+
+/**
+ * Gets a filtered list of Licenses from the SWS License service.
+ *
+ * Valid keys for the `$args` array provided to the constructor are:
```
Comment (sweetaz) [inline src/License/Command/LicenseList.php]: nbd, but maybe note that userid is required and the others are optional?

File: tests/License/Command/LicenseListTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+declare(strict_types=1);
+
+namespace Serato\SwsSdk\Test\License\Command;
+
+use Serato\SwsSdk\Test\AbstractTestCase;
+use Serato\SwsSdk\License\Command\LicenseList;
+
+class LicenseListTest extends AbstractTestCase
+{
+    public function testSmokeTest()
```
Comment (sweetaz) [inline tests/License/Command/LicenseListTest.php]: Do we need to add valid/invalid parameter tests here or are they covered elsewhere?

Comment (mshahidi) [reply tests/License/Command/LicenseListTest.php]: Yes, the tests for `Command` cover the required parameters.

File: src/License/Command/LicenseList.php (hunk 1 above)
Comment (mshahidi) [reply src/License/Command/LicenseList.php]: Sure, will change it

**Pull request merged**